ruby/metaparticle/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I'd like to add a code of conduct to the overall repository, and I'd like to use the Microsoft CoC:

https://opensource.microsoft.com/codeofconduct/

Let's do this as a separate PR?